README.md

```diff
@@ -119,21 +120,21 @@ for examples.
 
 ### Module template
 
-We have added a directory called [`software/SOFTWARE/TOOL/`](software/SOFTWARE/TOOL/) that serves as a template with which to create your own module submission. Where applicable, we have added extensive `TODO` statements to the files in this directory for general information, to help guide you as to where to make the appropriate changes, and how to make them. If in doubt, have a look at how we have done things for other modules.
+We have added a directory called [`software/TOOL/SUBTOOL/`](software/TOOL/SUBTOOL/) that serves as a template with which to create your own module and  [`tests/software/TOOL/SUBTOOL/`](tests/software/TOOL/SUBTOOL/) as an example of how to add the required CI tests. Where applicable, we have added extensive `TODO` statements for general information, to help guide you as to where to make the appropriate changes, and how to make them. If in doubt, have a look at how we have done things for other modules.
 
 ```console
 .
 ├── software
-│   └── SOFTWARE
-│       └── TOOL
-│           ├── functions.nf          ## Utility functions imported in main module script
-│           ├── main.nf               ## Main module script
-│           └── meta.yml              ## Documentation for module, input, output, params, author
-├── test
-│   └── SOFTWARE
-│       └── TOOL
-│           ├── main.nf           ## Minimal workflow to test module
-│           └── test.yml          ## Pytest-workflow test file
+│   └── TOOL
+│       └── SUBTOOL
+│           ├── functions.nf    ## Utility functions imported in main module script
+│           ├── main.nf         ## Main module script
+│           └── meta.yml        ## Documentation for module, input, output, params, author
+├── tests
+│   └── TOOL
+│       └── SUBTOOL
+│           ├── main.nf         ## Minimal workflow to test module
+│           └── test.yml        ## Pytest-workflow test file
```

Comment:
Currently we have `tests/software/tool` and also the `data` and `config` folders in the `tests`directory.
Are you planning to switch to this structure then? With `data` and `config` staying under the `tests` folder?

Just asking because of the linter :) although that will be quick to adapt to these changes.

Reply:
Yup. `data` and `config` will still stay under `tests` 👍🏽

Reply:
This path isn't right as it is right now, It should be `tests/software/TOOL/SUBTOOL/`